src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx

```diff
@@ -132,13 +151,28 @@ class ArtistPage extends React.Component {
 
         <PageTitle title={artist ? artist.attributes.name : '...'} context={'Apple Music'} />
         {geniusData && geniusData.plainDescription}
-        <h3>{translate.albums}</h3>
 
         {albums && (
-          <div className={classes.albumsGrid}>
-            {albums.map(album => (
-              <AlbumItem key={album.id} album={album} size={120} />
-            ))}
+          <div>
+            <h3>{translate.albums}</h3>
+
+            <div className={classes.albumsGrid}>
+              {albums.map(album => (
+                <AlbumItem key={album.id} album={album} size={120} />
+              ))}
+            </div>
+          </div>
+        )}
+
+        {playlists && (
+          <div>
```

Comment:
Away from my laptop this evening so can’t test right now. But does this need to be a div or could this be made a fragment (<>) instead? Don’t want to add anything unnecessary to the DOM

(Same for albums)

Reply:
Cool. I didn't knew about fragments. Thanks.

Reply:
No problem :) They’re a cool lil’ feature!

Thanks for your Pr though! Will test tomorrow before merging :)